src/groups/group_operation_action.jl

```diff
@@ -26,14 +26,8 @@ function switch_direction(A::GroupOperationAction)
     return GroupOperationAction(A.group, switch_direction(direction(A)))
 end
 
-apply(A::GroupOperationAction, a, p) = translate(A.group, a, p, direction(A))
```

Comment:
Why did you delete that?

Reply:
I've replaced it with `apply(A::GroupOperationAction, a, p) = error()` and the error is thrown in group_oeration_action.jl tests.

Reply:
And codecov reports it as covered.

Reply:
https://app.codecov.io/gh/JuliaManifolds/Manifolds.jl/blob/master/src/groups/group_operation_action.jl

Reply:
hm, then I do not know and lets undo this here. I was looking for test coverage I think and it was not covered and removing it, the tests still worked

Reply:
I restored that line.

Reply:
I meant `inverse_apply(A::GroupOperationAction, a, p)` in that other comment.

Tests still work because the mutating default is used.